docgen/parser/parser.go

```diff
@@ -26,14 +26,21 @@ const (
 	enumFlag    = 5
 )
 
+type OnlyExternalOpt int
```

Comment:
advantage over bool?

Reply:
The upside is it is clearer on function call: `func DoSomething(OnlyExternalEnabled)` vs `func DoSomething(true)`. The downside is we pollute the package with a new type.

Reply:
good point